docs/source/raft_ann_benchmarks.md

```diff
@@ -2,9 +2,26 @@
 
 This project provides a benchmark program for various ANN search implementations. It's especially suitable for comparing GPU implementations as well as comparing GPU against CPU.
 
-## Installing the benchmarks
+## Table of Contents
 
-The easiest way to install these benchmarks is through conda. We provide packages for GPU enabled systems, as well for systems without a GPU. We suggest using mamba as it generally leads to a faster install time:
+- [Installing and Running the Benchmarks](#installing--and-running-the-benchmarks)
+    - [Using conda](#conda)
+      - [End-to-end example: Million-scale](end-to-end-example-million-scale)
```

Comment:
It seems a little akward that we have the million-scale example embedded underneath "using conda" while we have the billion-scale as its own top-level. Should we move the million-scale to be just before the billion-scale?